### DISCUSSION TITLE
Strange behavior when using yarn dev --turbo in nextjs

### DISCUSSION THREAD

**Discussion by vitorGabr** [chakra-ui/panda#1593] [General]
When using nextjs and running it with the "--turbo" flag, I don't know if it's something related to turbopack or panda css, so I wanted help to solve it.

The errors are:

"error - src/app/global.css:3327:20  Parsing css source code failed"

"Error resolving EcmaScript Modules request
unable to resolve module "styled-system" with subpath "/jsx"
It was not possible to find the requested file.
     Parsed request as written in source code: module "styled-system" with subpath "/jsx"
     Path where resolving has started: [project]/src/app/loading.tsx
     Type of request: EcmaScript Modules request
     Import map: aliased to module "styled-system" with subpath "/jsx" inside of [project]/"

**Comment by astahmer** [comment]
hey, can you provide a minimal repro on a github repository ?

**Reply by vitorGabr** [reply]
@astahmer Of course here it is [repository,](https://github.com/vitorGabr/turbopack-test) as I said the problem is only when I run **"pnpm dev --turbo"**, when I run it without the flag it goes normal

**Reply by vitorGabr** [reply]
Any updates?

**Reply by astahmer** [reply]
seems like you can either:
- change your imports to be direct imports like `import { Center } from "styled-system/jsx/index.mjs";`
or use 
- `outExtension: "js",` in your panda.config (and run `panda codegen` again)

**Reply by vitorGabr** [reply]
Thank you very much, resolved!!
